test/ginkgo/testsuite_test.go

```diff
@@ -0,0 +1,101 @@
+// Before you run integration tests, you need to set
+// PULL_SECRET_LOCATION environment variable to point to your
+// pull-secret file. In case you are running a non-release binary
+// (with standalone bundle), you also need to set BUNDLE_LOCATION
+// variable to point to the location of the bundle you are using.
+
+package test_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+type VersionAnswer struct {
+	Version          string `json:"version"`
+	Commit           string `json:"commit"`
+	OpenshiftVersion string `json:"openshiftVersion"`
+	Embedded         bool   `json:"embedded"`
+}
+
+type StatusAnswer struct {
+	CRCStatus        string `json:"crcStatus"`
+	OpenshiftStatus  string `json:"openshiftStatus"`
+	OpenshiftVersion string `json:"openshiftVersion"`
+	DiskUsage        int    `json:"diskUsage"`
+	DiskSize         int    `json:"diskSize"`
+	CacheUsage       int    `json:"diskSize"`
+	CacheDir         string `json:"cacheDir"`
+}
+
+var credPath string
+var userHome string
+var versionInfo VersionAnswer
+
+var bundleLocation string
+var pullSecretLocation string
+
+func TestTest(t *testing.T) {
+
+	rand.Seed(time.Now().UTC().UnixNano()) // what's this for?
```

Comment:
This is needed if we use rand package and ask for random number. We don't use it in this PR, you can remove it!